cmd/cape/cmd/login.go

```diff
@@ -188,6 +194,23 @@ func getTokenResponse() (*TokenResponse, error) {
 	return &response, nil
 }
 
+func getAccessTokenVerifyAndParse() (jwt.Token, error) {
+	tokenResponse, err := getTokenResponse()
+	if err != nil {
+		return nil, err
+	}
+	var option = jwt.WithVerify(false)
+	// We cannot easily validate the JWKS in test as the test access token is self generated.
+	if viper.Get("GO_ENV") != "test" {
```

Comment:
@bendecoste Updated this to add JWKS validation. Also added a `GO_ENV` var to ensure we dont do this in test. Any other ideas or does this seem like an OK pattern here